docs/api-reference/mapbox/types.md

```diff
@@ -169,29 +169,6 @@ An object with the following fields:
 - `pitch`: number - The pitch (tilt) of the map, in degrees.
 - `bearing`: number - The bearing (rotation) of the map, in degrees.
 
-## Sources
-
-The following are implementations of source types that could be retrieved with `Map.getSource` ([Mapbox](https://docs.mapbox.com/mapbox-gl-js/api/map/#map#getsource) | [Maplibre](https://maplibre.org/maplibre-gl-js/docs/API/classes/Map/#getsource)).
-
-#### GeoJSONSource
-
-A source containing GeoJSON. See `GeoJSONSource` ([Mapbox](https://docs.mapbox.com/mapbox-gl-js/api/sources/#geojsonsource) | [Maplibre](https://maplibre.org/maplibre-gl-js/docs/API/classes/GeoJSONSource/)).

```

Comment:
Is there any difference between the Mapbox style spec and the API doc contents for these Source types? (I see how they're duped above, except for the links)